Flow.Launcher/SettingPages/Views/SettingsPaneTheme.xaml.cs

```diff
@@ -28,4 +30,52 @@ private void Selector_OnSelectionChanged(object sender, SelectionChangedEventArg
     {
         _viewModel.UpdateColorScheme();
     }
+
+    private void Reset_Click(object sender, RoutedEventArgs e)
+    {
+        /*The FamilyTypeface should initialize all of its various properties.*/
+        FamilyTypeface targetTypeface = new FamilyTypeface { Stretch = FontStretches.Normal, Weight = FontWeights.Normal, Style = FontStyles.Normal };
+
+        QueryBoxFontSize.Value = 20;
+        QueryBoxFontComboBox.SelectedIndex = SearchFontIndex("Segoe UI", QueryBoxFontComboBox);
+        QueryBoxFontStyleComboBox.SelectedIndex = SearchFontStyleIndex(targetTypeface, QueryBoxFontStyleComboBox);
+
+        ResultItemFontComboBox.SelectedIndex = SearchFontIndex("Segoe UI", ResultItemFontComboBox);
+        ResultItemFontStyleComboBox.SelectedIndex = SearchFontStyleIndex(targetTypeface, ResultItemFontStyleComboBox);
+        ResultItemFontSize.Value = 16;
+
+        ResultSubItemFontComboBox.SelectedIndex = SearchFontIndex("Segoe UI", ResultSubItemFontComboBox);
+        ResultSubItemFontStyleComboBox.SelectedIndex = SearchFontStyleIndex(targetTypeface, ResultSubItemFontStyleComboBox);
+        ResultSubItemFontSize.Value = 13;
+
+        WindowHeightValue.Value = 42;
```

Comment:
why there's a `.Value` ? shouldn't it be an int? just took a quick glance and don't have time to further review it.

Reply:
This item changes the value of a specific control (slider) rather than a setting. 
This was implemented because there was an issue with changing the value of a set value not being reflected in the control.

Reply:
I think you may need double binding if wanting to change the setting instead of the control value.